automatic/firefox/tools/chocolateyInstall.ps1

```diff
@@ -17,26 +19,65 @@ if (Get-32bitOnlyInstalled -product $softwareName) {
   )
 }
 
+$args = ""
+
+# Command Line Options from the Firefox installer
+# https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html
+
+# Always prevent Firefox installer to require a reboot
+$args = $args + " /PreventRebootRequired=true"
+
+# Prevent RemoveDistributionDir by default
+$args = $args + " /RemoveDistributionDir=false"
+
+
+if ($PackageParameters['InstallDir']) {
```

Comment:
There is a 1 line form: 

- `$sa += if ($pp.NoTaskbar) { " /RemoveDistributionDir=false" }`


Also, change `$args`, to some other var name, its used by powershell for function args.

Reply:
Great, thanks for this hint. It's implemented now.